src/app.js

```diff
@@ -1,13 +1,186 @@
 // Vendor Modules
 import $ from 'jquery';
 import _ from 'underscore';
+import 'jquery-modal';
 
 // CSS
 import './css/foundation.css';
 import './css/style.css';
 
-console.log('it loaded!');
+// Import Trip Model and Collection
+import Trip from './app/models/trip';
+import TripList from './app/collections/trip_list';
+import Reservation from './app/models/reservation';
+
+const tripFields = ['name', 'continent', 'about', 'category', 'weeks', 'cost'];
+const reservationFields = ['name', 'age', 'email'];
+
+const events = {
+  sortTrips(event){
+    $('.sort').removeClass('current-sort-field');
+    $(this).addClass('current-sort-field');
+    let classes = $(this).attr('class').split(/\s+/);
+    classes.forEach((className) => {
+      if(tripFields.includes(className)){
+        if (className === tripList.comparator){
+          tripList.models.reverse();
```

Comment:
This is a temporary change -- Backbone won't "know" that we've reversed the sort order for this comparator.

As a result, when Backbone decides to re-sort the collection for us (such as when a new trip is added), it will go back to the non-reversed order. How could we implement the sorting for our Backbone collection such that it will keep track of whether the sort was reversed or not?